content/blog/tyranny-of-nits/index.md

```diff
@@ -37,50 +48,91 @@ It's really tempting, and really easy, to just blame the reviewer here.
 If they just chilled, just let the nits slide, just fixed them themselves, didn't scope creep: then everything would be okay!
 
 But that's not a real solution for anyone other than "the nitpicking reviewer".
-Around here, **we believe in systematic solutions to systematic problems.**
-And if reading this post has left you with as many bitter memories of [great ideas that may never be](https://soasis.org/posts/a-mirror-for-rust-a-plan-for-generic-compile-time-introspection-in-rust/) as writing it left in me,
-you can be assured this is a systematic problem.
+Why does questionable but well-intenioned behavior quietly derail important work?
+
+## Overruling objections: naive consensus is terrible actually
+
+To understand this, we need to examine the decision making process of open source projects, as they *actually* exist.
+Once a PR is opened, what happens to it?
+
+Open source projects tend to run on **naive consensus by default.**
+The operating semantics are simple:
+
+1. If everyone says "LGTM", the work gets merged.
+2. If someone reasis a concern, we wait for the author to fix it.
+3. If the author doesn't fix the concern, the PR stays open (or gets closed as stale).
+4. If the author comes back and says "that's not a problem actually", then we get orgnizational undefined behavior.
+
+We even made a [witty, heart-breaking game](
+https://leafwing-studios.itch.io/consensus-together) you can play with your friends/teammates/worst enemies, if you don't believe that naive consensus itself is at fault here.
+
+These rules make open source pretty good at saying yes to uncontroversial things,
+and extremely good at saying no to controversial things, because a conflict-driven stalemate or a timeout due to excessive revisions is still a no.
+
+But that's a *problem*: you have to be able to say yes to controversial decisions too!
+Axiomatically: to make a decision, you have to both be able to say no, and to say yes.
+If your project works like this, your project **can't make decisions about the things that matter.**
+You have to keep making decisions, day after day, or your momentum will die and everything will slowly rot.
+It doesn't even matter if they're "right": the very fact that a decision was made staves off this decay.
+
+The classical solution here is to have a BDFL, or perhaps a code owner, who can shut down discussion, merge PRs over objections and close issues as "not planned".
+As long as they're not overworked (never happens), things function pretty smoothly.
+
+But that's not the only patch: delegation, binding votes or [more sophisticated consensus-based decision making processes](https://www.sociocracyforall.org/sociocracy/) all offer another way.
+Debate these and experiment: I think any of them can work just fine.
+Just don't ignore this problem!
 
-## Automation for humans
+## Nit-pick arithme-tick
```

Comment:
I'd add a sentence at the start here about how endless nitpick cycles technically stave off "deadlock by controversy", but still kill momentum in their own way. The last section ended with what sounded like solutions, so why are we still talking?